lib/rules/lint-table-groups.js

```diff
@@ -24,13 +24,18 @@ function hasDisallowedChildren(tableNode) {
       childrenFromValidBlocks,
       tableNode.children.filter(item => validBlocks.indexOf(item) === -1)
     )
-    .some(
-      child =>
-        !(
-          (child.type === 'ElementNode' && ALLOWED_TABLE_CHILDREN.indexOf(child.tag) > -1) ||
-          (child.type === 'TextNode' && !/\S/.test(child.chars))
-        )
-    );
+    .some(child => {
+      if (child.type === 'MustacheStatement') {
```

Comment:
At this point, I think it would be cleaner to do a `switch` on `child.type`.

Reply:
updated